src/Presentation/Console/Command/Validator/ValidateTaskCommand.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Presentation\Console\Command\Validator;
+
+use SprykerSdk\Sdk\Core\Application\Dependency\Repository\TaskYamlRepositoryInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ValidateTaskCommand extends Command
+{
+    /**
+     * @var string
+     */
+    protected const NAME = 'sdk:validate:task';
+
+    /**
+     * @var \SprykerSdk\Sdk\Core\Application\Dependency\Repository\TaskYamlRepositoryInterface
+     */
+    private TaskYamlRepositoryInterface $taskYamlRepository;
+
+    /**
+     * @param \SprykerSdk\Sdk\Core\Application\Dependency\Repository\TaskYamlRepositoryInterface $taskYamlRepository
+     */
+    public function __construct(TaskYamlRepositoryInterface $taskYamlRepository)
+    {
+        $this->taskYamlRepository = $taskYamlRepository;
+
+        parent::__construct(static::NAME);
+    }
+
+    /**
+     * @param \Symfony\Component\Console\Input\InputInterface $input
+     * @param \Symfony\Component\Console\Output\OutputInterface $output
+     *
+     * @return int
+     */
+    public function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $this->taskYamlRepository->findAll();
```

Comment:
This is completely wrong. You should've validated a specific task. Functional requirements mismatch. Please, change it.

Reply:
everything correct.  it's wrong that `findAll` returns DTO without validation. in this case wrong structure can break logic. that's why was decided to use https://symfony.com/doc/current/components/config/definition.html